packages/mdc-select/README.md

```diff
@@ -129,6 +129,13 @@ style dependencies for both the mdc-list and mdc-menu for this component to func
 </div>
 ```
 
+#### Preventing [FOUC](https://en.wikipedia.org/wiki/Flash_of_unstyled_content)
```

Comment:
Oops, looks like I fat-fingered the markdown here - meant to nest the `Preventing FOUC` section under the `Select with pre-selected option`. 

Should I fix the nesting so that it provides context to the `Select with pre-selected option` section, or just remove the `Preventing FOUC` section altogether?

Reply:
Change it to 
``` 
When dealing with the select component that has pre-selected values, you'll want to ensure that you
render `mdc-select__label` with the `mdc-select__label--float-above` modifier class and the selected option with `aria-selected`. This will
ensure that the label moves out of the way of the select's value and prevents a Flash Of
Un-styled Content (**FOUC**).
```
so that it more closely matches the text field explanation for pre-filled values. Also, it should be above the example on Line 99.